pygmt/src/blockm.py

```diff
@@ -73,6 +134,30 @@ def blockmedian(table, outfile=None, **kwargs):
         - None if ``outfile`` is set (filtered output will be stored in file
           set by ``outfile``)
     """
+    return _blockm(block_method="blockmedian", table=table, outfile=outfile, **kwargs)
+
+
+def _blockm(block_method, table, outfile, **kwargs):
+    r"""
+    Block average (x,y,z) data tables by mean or median estimation.
+
+    Reads arbitrarily located (x,y,z) triples [or optionally weighted
+    quadruples (x,y,z,w)] from a table and writes to the output a mean or
+    median (depending on ``block_method``) position and value for every
+    non-empty block in a grid region defined by the ``region`` and ``spacing``
+    parameters.
```

Comment:
Add a "Parameters" section and explain the "block_method" parameter.

Reply:
Added in https://github.com/GenericMappingTools/pygmt/pull/1092/commits/62a763bc68a91e2521e9c77db74cd55696335f7a